src/sqlfluff/core/dialects/dialect_ansi.py

```diff
@@ -746,6 +747,16 @@ class FrameClauseSegment(BaseSegment):
 )
 
 
+class AliasInfo(NamedTuple):
```

Comment:
In working on this PR, I refactored some of these functions from using tuples to using NamedTuples. It makes the code easier to understand and maintain.